modules/renderer/map.js

```diff
@@ -217,8 +217,6 @@ export function Map(context) {
     function redraw(difference, extent) {
         if (!surface || !redrawEnabled) return;
 
-        clearTimeout(timeoutId);
```

Comment:
i think you're right. In practice this would only do one more redraw call (from the panning) after the difference/extent call and would still be a perf improvement when comparing to the previous implementation. I'll dig in some more later and do an amendment to the PR.